.github/workflows/update-dependencies.yml

```diff
@@ -3,7 +3,7 @@ name: Update python dependencies
 on:
   workflow_dispatch:
   schedule:
-    - cron:  "0 23 * * *"
+    - cron:  "0 23 * * WED"
```

Comment:
We've got some repos running early (04:00) on Wed and some running late (23:00) so we don't actually see the notification until Thurs (entirely my fault). I think we probably want them all to be early?